src/docs/introduction/upgrading-to-stencil-three.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Upgrading to Stencil v3.0.0
+description: Upgrading to Stencil v3.0.0
+url: /docs/upgrading-to-stencil-3
+contributors:
+  - rwaskiewicz
+---
+
+# Upgrading to Stencil v3.0.0
+
+## Getting Started
+
+It's recommended that your projects start their upgrade from Stencil v2.
+Projects using Stencil v0 or Stencil v1 should upgrade to Stencil v2 before proceeding with this guide.
+For breaking changes introduced in previous major versions of the library, see:
+- [Stencil v2 Breaking Changes](https://github.com/ionic-team/stencil/blob/main/BREAKING_CHANGES.md#stencil-two)
```

Comment:
I didn't use permalinks here, as GitHub doesn't allow you to access a markdown file through a permalink _and_ render it in the "pretty" rich-diff view. I can switch this, but from an aesthetic perspective, I liked being sent to the rich view.